### ISSUE TITLE
How to delete sqlite database

### ISSUE THREAD

**Issue by koseduhemak** [nextcloud/android#6109]
Hi there,

I was wondering if it is possible to delete the sqlite database the Android app uses.
If I uninstall the app / clear cache / clear storage Nextcloud app still remembers / cached my folder structure after reinstalling the app. If I open the app after reinstall, I see my "old" files, which I already removed. The app then tries to sync but crashes with exception:
Because there was a very large folder, the app crashes (SQLiteToobig... Exception).

I installed the Nextcloud dev app vai F-Droid which is now working fine (as it is reading the file structure from scratch).

Any help appreciated.

Thanks

**Comment by tobiasKaminsky** [comment]
When you remove the app it should delete all app specific data.
Do you maybe have some kind of backup mechanism? 
There is an option to restore data via Android/Google?

**Comment by koseduhemak** [comment]
Thanks for your reply.
The hint with the backup mechanism was gold!
I didn‘t know that Google does automatically backup data of apps into Google Drive. This was causing the error as it restored the „corrupted“ sqllite database from there anytime I reinstalled the app. I manually deleted the backup in Google Drive and now everything works as expected.

Things are complicated these days....

Thank you for your help!
